db/migrate/20210317030546_add_title_reference_to_student_table.rb

```diff
@@ -0,0 +1,9 @@
+class AddTitleReferenceToStudentTable < ActiveRecord::Migration
+  def change
+    # add foreign key title_id column
+     change_table :students do |t|
+      t.references :title, null: false
```

Comment:
Is it intended to force students to have a `title`?

Reply:
A few PRs back Grant recommended adding `null: false` to columns to safeguard against any null error issues later on.